src/lib/errors/CombinedError.ts

```diff
@@ -5,7 +5,7 @@ export class CombinedError extends BaseError {
 	public readonly errors: readonly BaseError[];
 
 	public constructor(errors: readonly BaseError[]) {
-		super('Received one or more errors');
+		super(errors.length === 1 ? errors[0].message : 'Received multiple errors');
```

Comment:
```suggestion
		super(errors.length === 1 ? errors[0].message : `Received ${errors.length} errors`);
```
and if u r changing it then we can directly pass the number now